babel.config.js

```diff
@@ -1,4 +1,18 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
-  plugins: ['react-native-reanimated/plugin'],
+  plugins: [
+    [
```

Comment:
1. Added safe react-native-dotenv config
2. Added react-native-reanimated config which according to its README should be placed at the bottom in all plugins